### PR TITLE
Add section describing the the relationship between Solid Protocol and LDP

### PR DESCRIPTION
**Pull request by csarven**
Adds a non-normative section ( [#relationship-to-ldp](http://htmlpreview.github.io/?https://github.com/solid/specification/blob/01d409b3f0e7eb8f28ffe2738bf4c365e06dd9f9/ED/protocol.html#relationship-to-ldp) ) under Introduction describing the the relationship between Solid Protocol and LDP. This is a https://www.w3.org/2023/Process-20230612/#class-2 change. See [#changelog](http://htmlpreview.github.io/?https://github.com/solid/specification/blob/01d409b3f0e7eb8f28ffe2738bf4c365e06dd9f9/ED/protocol.html#changelog) .

Resolves https://github.com/solid/specification/issues/224 . Any additional information about the relationship between the Solid Protocol and LDP can be PR'd (optionally with more specific issues targeting the differences/similarities that needs to be clarified).

This PR also follows the recommendation of W3C Process regarding explaining "how the technology relates to existing international standards and related work inside or outside W3C" as per  https://www.w3.org/2023/Process-20230612/#publication . It is intended to help W3C horizontal reviewers ( e.g., https://github.com/solid/solid-wg-charter/issues/15 ) as well as anyone already familiar with either Solid Protocol or LDP (or both) to understand where they come together or diverge.

Certainly more can be said about the Solid Protocol / LDP. Reviewers of this PR should take this as initial work rather than complete. This text is derived from https://github.com/solid/specification/issues/194#issuecomment-694828342 based on rough consensus.

---

[Preview](http://htmlpreview.github.io/?https://github.com/solid/specification/blob/01d409b3f0e7eb8f28ffe2738bf4c365e06dd9f9/ED/protocol.html) | [Diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fspecification%2F8823c48aa7b01c742689dc49dbd0ebe749cdebee%2FED%2Fprotocol.html&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fspecification%2F01d409b3f0e7eb8f28ffe2738bf4c365e06dd9f9%2FED%2Fprotocol.html)

### CODE REVIEW

**Review by woutermont — APPROVED** [review]
Not entirely sure of the "Relations" parent section, but the text is great!

File: ED/protocol.html [hunk 1]
```diff
@@ -630,6 +631,30 @@ <h4 property="schema:name skos:prefLabel">Interoperability</h4>
...
+
+                      <p>As HTTP clients assign a URI to a resource when using <code>PUT</code> and <code>PATCH</code>, relative references in representation content are deterministic in both the Solid Protocol and LDP.</p>
+
+                      <p>LDP does not specify URI patterns for resources when using the <code>POST</code> method, e.g., <code>POST http://example.org/foo/</code> may result in creating a resource with URI <code>http://example.org/foo/bar</code> , <code>http://example.org/baz/qux/quxx</code>, <code>http://example.org/{uuid}</code> or something else. In Solid Protocol, the URI of a new resource resulting from, e.g., <code>POST http://example.org/foo/</code>, is in context of the request URI, and therefore <code>http://example.org/foo/bar</code> will be created, and it is not possible to construct the new URI besides under the direct hierarchy of <code>http://example.org/foo/</code>.</p>
```
Comment (TallTed) [inline ED/protocol.html:649]:
```suggestion
                      <p>LDP does not specify URI patterns for resources when using the <code>POST</code> method, e.g., <code>POST http://example.org/foo/</code> may result in creating a resource with URI <code>http://example.org/foo/bar</code>, <code>http://example.org/baz/qux/quxx</code>, <code>http://example.org/{uuid}</code> or something else. In Solid Protocol, the URI of a new resource resulting from, e.g., <code>POST http://example.org/foo/</code>, is in context of the request URI, and therefore <code>http://example.org/foo/bar</code> will be created, and it is not possible to construct the new URI besides under the direct hierarchy of <code>http://example.org/foo/</code>.</p>
```

File: ED/protocol.html [hunk 2]
```diff
@@ -630,6 +631,30 @@ <h4 property="schema:name skos:prefLabel">Interoperability</h4>
...
+
+                      <p>Solid Protocol <a href="#Client">clients</a> may not fully interoperate with LDP servers, e.g., it is possible for an LDP server implementation to assign URIs to new resources that is not compatible with Solid Protocol's. An LDP server needs to also conform to Solid Protocol server requirements in order to participate in the Solid ecosystem.</p>
```
Comment (TallTed) [inline ED/protocol.html:653]:
```suggestion
                      <p>Solid Protocol <a href="#Client">clients</a> may not fully interoperate with LDP servers, e.g., it is possible for an LDP server implementation to assign URIs to new resources that are not compatible with Solid Protocol. An LDP server needs to also conform to Solid Protocol server requirements in order to participate in the Solid ecosystem.</p>
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Just a couple of tweaks for clarity

**Review by kjetilk — APPROVED** [review]
It has been so long that I don't remember the details anymore. There has been quite a lot of tension...

Nevertheless, I believe this would help clarify and so even if more issues are found later, it might well go in now.

**Comment by csarven** [comment]
Merging as per agreement in 2023-07-12 meeting: https://github.com/solid/specification/blob/main/meetings/2023-07-12.md#add-section-describing-the-the-relationship-between-solid-protocol-and-ldp

**Pull request merged**